jumpcloud-ADMU/Powershell/Form.ps1

```diff
@@ -661,13 +661,32 @@ $bMigrateProfile.Add_Click( {
             } else {
                 Write-ToLog "ConnectKey is populated, JumpCloud agent will be installed"
             }
-            $testResult, $userID = Test-JumpCloudUsername -JumpCloudApiKey $tbJumpCloudAPIKey.Text -JumpCloudOrgID $lbl_orgId.Text -Username $tbJumpCloudUserName.Text -Prompt $true
+
+            $testResult, $JumpCloudUserId, $jcUsername, $JCSystemUsername = Test-JumpCloudUsername -JumpCloudApiKey $tbJumpCloudAPIKey.Text -JumpCloudOrgID $lbl_orgId.Text -Username $tbJumpCloudUserName.Text -Prompt $true
             if ($testResult) {
                 Write-ToLog "Matched $($tbJumpCloudUserName.Text) with user in the JumpCloud Console"
             } else {
                 Write-ToLog "$($tbJumpCloudUserName.Text) not found in the JumpCloud console"
                 return
             }
+
+
+            if ($JCSystemUsername) {
+                Write-ToLog "User $($JCSystemUsername) has a local account on this system"
+                $wshell = New-Object -ComObject Wscript.Shell
+                $message = "Would you like to migrate the local user profile to the JumpCloud User $($jcsystemUserName)?"
+                $var = $wshell.Popup("$message", 0, "JC Local User Check", 64 + 4)
+                # If user selects yes then migrate the local user profile to the JumpCloud User
+
+                if ($var -eq 6) {
+                    Write-ToLog -Message "User selected Yes, continuing with migration $($jcsystemUserName))"
+                } else {
+                    Write-ToLog -Message "User selected No, returning to form"
+                    return
+                }
+            } else {
+                Write-ToLog "User $($jcsystemUserName) does not have a local account on this system"
```

Comment:
```suggestion
                Write-ToLog "User $($jcUsername) does not have a local account username"
```

Reply:
If we are testing for `$jcsystemUserName` in the IF block before this Else block, then we expect `$jcsystemUserName` to be null, the resulting log message would be " does not have a local account on this system" which doesn't provide any useful information. 

Instead, we should be using the JumpCloud username that we identified with `Test-JumpCloudUsername`